src/univers/version_constraint.py

```diff
@@ -0,0 +1,221 @@
+#
+# Copyright (c) nexB Inc. and others.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Visit https://aboutcode.org and https://github.com/nexB/univers for support and download.
+
+import operator
+from functools import total_ordering
+
+import attr
+
+from univers.utils import remove_spaces
+from univers.versions import Version
+
+"""
+Universal version constraint object that stores a comparator such as "=" and
+an ecosystem- or package-specific Version object.
+"""
+
+
+def operator_star(a, b):
```

Comment:
I can't imagine where this is used. A version range like "*1.2.3" is unlikely exist. However stars are commonly used like "1.2.*" .

Reply:
Oh, I see what you mean from the below doc test.

Reply:
I am not 100% happy with this start thing... but the intention is to have an explicit range that means "any version" rather than having no range for this case. Otherwise, there is no easy way for this beyond having no range?